packages/repo/tests/mst.test.ts

```diff
@@ -155,378 +155,273 @@ describe('Merkle Search Tree', () => {
     }
   })
 
-  // Special Cases (these are made for fanout 32)
-  // ------------
```

Comment:
I know these special cases were specific to the fanout— will we end-up recreate new versions of them for the new fanout?

Reply:
Yup the interop stuff at the bottom of the test file is the exact same tests but targeted for fanout 4 👍